metadata-ingestion-modules/gx-plugin/setup.py

```diff
@@ -0,0 +1,163 @@
+import os
+import pathlib
+
+import setuptools
+
+package_metadata: dict = {}
+with open("./src/datahub_gx_plugin/__init__.py") as fp:
+    exec(fp.read(), package_metadata)
+
+
+def get_long_description():
+    root = os.path.dirname(__file__)
+    return pathlib.Path(os.path.join(root, "README.md")).read_text()
+
+
+rest_common = {"requests", "requests_file"}
+
+# TODO: Is this required here??
+pydantic_no_v2 = {
+    # pydantic 2 makes major, backwards-incompatible changes - https://github.com/pydantic/pydantic/issues/4887
+    # Tags sources that require the pydantic v2 API.
+    "pydantic<2",
+}
+# TODO: Can we move away from sqllineage and use sqlglot ??
```

Comment:
ideally yes. let's keep this as a follow up though, so that this PR is purely refactoring

Reply:
Keeping this TODO as is.